lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/sys.rb

```diff
@@ -37,7 +37,7 @@ class Console::CommandDispatcher::Stdapi::Sys
     "-p" => [ false, "Execute process in a pty (if available on target platform)"	   ],
     "-s" => [ true,  "Execute process in a given session as the session user"  ])
 
-  @@execute_opts_with_raw_mode = Rex::Parser::Arguments.new(@@execute_opts.fmt.merge(
+  @@execute_opts_with_raw_mode = Rex::Parser::Arguments.new(@@execute_opts.merge(
     { '-r' => [ false, 'Raw mode'] }
   ))
```

Comment:
Potentially a better API:

```suggestion
  @@execute_opts_with_raw_mode = @@execute_opts.merge(
    { '-r' => [ false, 'Raw mode'] }
  )
```